fdbserver/CommitProxyServer.actor.cpp

```diff
@@ -1278,11 +1302,19 @@ ACTOR Future<Void> reply(CommitBatchContext* self) {
 	// self->committedVersion.
 	TEST(pProxyCommitData->committedVersion.get() > self->commitVersion); // A later version was reported committed first
 	if (self->commitVersion >= pProxyCommitData->committedVersion.get()) {
+		state Optional<VersionVector> ssVersionVector;
+		if (SERVER_KNOBS->ENABLE_VERSION_VECTOR) {
+			ssVersionVector = VersionVector();
+			for (Tag tag : self->tagSet) {
+				ssVersionVector.get().setVersion(tag, self->commitVersion);
```

Comment:
Since we are already propagating "self->commitVersion" as part of the message anyway, isn't it better to send the tags as "std::set<Tag", instead of as a version vector?

Reply:
done